### ISSUE TITLE
how to load pt file

### ISSUE THREAD

**Issue by kase113** [dem123456789/HeteroFL-Computation-and-Communication-Efficient-Federated-Learning-for-Heterogeneous-Clients#6]
If i have trained a new model,and get a 'pt' file.Can i how to load this pt file?

**Comment by dem123456789** [comment]
The 'pt' extension does not matter much. It is a acronym convention of Pytorch. You can use the method in ./utils.py to load the pt file. It is a dictionary contains model, optimizer and scheduler state_dict().
